client/state/plugins/installed/test/actions.js

```diff
@@ -0,0 +1,554 @@
+/**
+ * External dependencies
+ */
+import nock from 'nock';
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	PLUGINS_RECEIVE,
+	PLUGINS_REQUEST,
+	PLUGINS_REQUEST_SUCCESS,
+	PLUGINS_REQUEST_FAILURE,
+	PLUGIN_ACTIVATE_REQUEST,
+	PLUGIN_ACTIVATE_REQUEST_SUCCESS,
+	PLUGIN_ACTIVATE_REQUEST_FAILURE,
+	PLUGIN_DEACTIVATE_REQUEST,
+	PLUGIN_DEACTIVATE_REQUEST_SUCCESS,
+	PLUGIN_DEACTIVATE_REQUEST_FAILURE,
+	PLUGIN_UPDATE_REQUEST,
+	PLUGIN_UPDATE_REQUEST_SUCCESS,
+	PLUGIN_UPDATE_REQUEST_FAILURE,
+	PLUGIN_AUTOUPDATE_ENABLE_REQUEST,
+	PLUGIN_AUTOUPDATE_ENABLE_REQUEST_SUCCESS,
+	PLUGIN_AUTOUPDATE_ENABLE_REQUEST_FAILURE,
+	PLUGIN_AUTOUPDATE_DISABLE_REQUEST,
+	PLUGIN_AUTOUPDATE_DISABLE_REQUEST_SUCCESS,
+	PLUGIN_AUTOUPDATE_DISABLE_REQUEST_FAILURE,
+	PLUGIN_INSTALL_REQUEST,
+	PLUGIN_INSTALL_REQUEST_SUCCESS,
+	PLUGIN_INSTALL_REQUEST_FAILURE,
+	PLUGIN_REMOVE_REQUEST,
+	PLUGIN_REMOVE_REQUEST_SUCCESS,
+	PLUGIN_REMOVE_REQUEST_FAILURE
+} from 'state/action-types';
+import {
+	INSTALL_PLUGIN,
+	REMOVE_PLUGIN,
+	UPDATE_PLUGIN,
+	ACTIVATE_PLUGIN,
+	DEACTIVATE_PLUGIN,
+	ENABLE_AUTOUPDATE_PLUGIN,
+	DISABLE_AUTOUPDATE_PLUGIN
+} from '../constants';
+import {
+	fetch as fetchPlugins,
```

Comment:
Another point to why it'd be nice if the renamed import here was simply the original name of the action creator 😄